### PR TITLE
[OCPBUGS-6849]: Update commands to find etcd pods in OCP 4.10

### PR DESCRIPTION
**Pull request by tmalove**
[OCPBUGS-6849](https://issues.redhat.com/browse/OCPBUGS-6849)
** Reviewers: This PR is to CP this update to the 4.10 branch. The original Jira applied to releases 4.9+, but this update was completed separately.

Version(s): 4.10

Links to docs preview:
 - Restoring to a previous cluster state (Steps 11b and 18): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp410-tlove/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state
 - Manual defragmentation (step 1a): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp410-tlove/scalability_and_performance/recommended-host-practices.html#manual-defrag-etcd-data_recommended-host-practices
 - Determining the state of the unhealthy etcd member (step 3b): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp410-tlove/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-determine-state-etcd-member_replacing-unhealthy-etcd-member
 - Replacing an unhealthy etcd member whose etcd pod is crashlooping (step 2a): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp410-tlove/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-crashlooping-etcd-member_replacing-unhealthy-etcd-member
 - Replacing an unhealthy bare metal etcd member whose machine is not running or whose node is not ready (step 1a): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp410-tlove/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-baremetal-etcd-member_replacing-unhealthy-etcd-member
 - Replacing an unhealthy etcd member whose machine is not running or whose node is not ready (step 1a and verification, step 1): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp410-tlove/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-etcd-member_replacing-unhealthy-etcd-member

QE review:
- [ x] QE has approved this change (Ge Liu).

### CODE REVIEW

**Comment by tmalove** [comment]
Hi @wking  Review request: this PR is for release 4.10. I made the changes directly in the release. Thanks!

File: modules/etcd-defrag.adoc [hunk 1]
```diff
@@ -87,7 +87,7 @@ Follow this procedure to defragment etcd data on each etcd member.
 +
 [source,terminal]
 ----
-$ oc get pods -n openshift-etcd -o wide | grep -v quorum-guard | grep etcd
+$ oc -n openshift-etcd get pods -l k8s-app=etcd
```
Comment (wking) [inline modules/etcd-defrag.adoc:90]: this one should preserve the `-o wide` from the outgoing request

File: modules/restore-replace-stopped-baremetal-etcd-member.adoc [hunk 2]
```diff
@@ -570,7 +570,7 @@ In a terminal that has access to the cluster as a `cluster-admin` user, run the
 +
 [source,terminal]
 ----
-$ oc get pods -n openshift-etcd -o wide | grep etcd | grep -v guard
+$ oc -n openshift-etcd get pods -l k8s-app=etcd
```
Comment (wking) [inline modules/restore-replace-stopped-baremetal-etcd-member.adoc:573]: This one should preserve the outgoing `-o wide`

**Comment by tmalove** [comment]
@wking I made the latest update for your review. Thanks!

**Comment by tmalove** [comment]
@geliu2016 Will you review this update? Thanks!

**Comment by kelbrown20** [comment]
@tmalove Hey Tami! Could you squash those commits before merging?

**Comment by tmalove** [comment]
@kelbrown20 I just squashed the commits...thanks!

**Comment by tmalove** [comment]
@kelbrown20  Should I ask Steven to set it again?

**Pull request merged**